Some indirect dependency vulnerabilities have been detected

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!

-->

=== Symbol Results ===

Vulnerability #1: GO-2025-3563
    Request smuggling due to acceptance of invalid chunked data in net/http
  More info: https://pkg.go.dev/vuln/GO-2025-3563
  Standard library
    Found in: net/http/internal@go1.24
    Fixed in: net/http/internal@go1.24.2
    Example traces found:
      #1: runtime/sys.go:207:32: runtime.readLinesFromFile calls bufio.Reader.ReadLine, which eventually calls internal.chunkedReader.Read

Vulnerability #2: GO-2024-2687
    HTTP/2 CONTINUATION flood in net/http
  More info: https://pkg.go.dev/vuln/GO-2024-2687
  Module: golang.org/x/net
    Found in: golang.org/x/net@v0.9.0
    Fixed in: golang.org/x/net@v0.23.0
    Example traces found:
      #1: database/kv/etcd/v3/client.go:30:2: etcd.init calls client.init, which eventually calls http2.ConnectionError.Error
      #2: container/gxlru/lru_cache.go:199:20: gxlru.LRUCache.StatsJSON calls fmt.Sprintf, which eventually calls http2.ErrCode.String
      #3: container/gxlru/lru_cache.go:199:20: gxlru.LRUCache.StatsJSON calls fmt.Sprintf, which eventually calls http2.FrameHeader.String
      #4: container/gxlru/lru_cache.go:199:20: gxlru.LRUCache.StatsJSON calls fmt.Sprintf, which eventually calls http2.FrameType.String
      #5: database/kv/etcd/v3/client.go:137:32: etcd.NewClientWithOptions calls client.New, which eventually calls http2.Framer.ReadFrame
      #6: database/kv/etcd/v3/client.go:137:32: etcd.NewClientWithOptions calls client.New, which eventually calls http2.Framer.WriteContinuation
      #7: database/kv/etcd/v3/client.go:137:32: etcd.NewClientWithOptions calls client.New, which eventually calls http2.Framer.WriteData
      #8: database/kv/etcd/v3/client.go:137:32: etcd.NewClientWithOptions calls client.New, which eventually calls http2.Framer.WriteHeaders
      #9: database/kv/etcd/v3/client.go:137:32: etcd.NewClientWithOptions calls client.New, which eventually calls http2.Framer.WritePing
      #10: database/kv/etcd/v3/client.go:137:32: etcd.NewClientWithOptions calls client.New, which eventually calls http2.Framer.WriteRSTStream
      #11: database/kv/etcd/v3/client.go:137:32: etcd.NewClientWithOptions calls client.New, which eventually calls http2.Framer.WriteSettings
      #12: database/kv/etcd/v3/client.go:137:32: etcd.NewClientWithOptions calls client.New, which eventually calls http2.Framer.WriteSettingsAck
      #13: database/kv/etcd/v3/client.go:137:32: etcd.NewClientWithOptions calls client.New, which eventually calls http2.Framer.WriteWindowUpdate
      #14: database/kv/etcd/v3/client.go:30:2: etcd.init calls client.init, which eventually calls http2.GoAwayError.Error
      #15: container/gxlru/lru_cache.go:199:20: gxlru.LRUCache.StatsJSON calls fmt.Sprintf, which eventually calls http2.Setting.String
      #16: container/gxlru/lru_cache.go:199:20: gxlru.LRUCache.StatsJSON calls fmt.Sprintf, which eventually calls http2.SettingID.String
      #17: database/kv/etcd/v3/client.go:137:32: etcd.NewClientWithOptions calls client.New, which eventually calls http2.SettingsFrame.ForeachSetting
      #18: database/kv/etcd/v3/client.go:30:2: etcd.init calls client.init, which eventually calls http2.StreamError.Error
      #19: log/color.go:100:14: log.CEPrintfln calls fmt.Fprintf, which eventually calls http2.chunkWriter.Write
      #20: database/kv/etcd/v3/client.go:30:2: etcd.init calls client.init, which eventually calls http2.connError.Error
      #21: database/kv/etcd/v3/client.go:30:2: etcd.init calls client.init, which eventually calls http2.duplicatePseudoHeaderError.Error
      #22: database/kv/nacos/config_client.go:57:40: nacos.NacosConfigClient.newConfigClient calls clients.NewConfigClient, which eventually calls http2.gzipReader.Close
      #23: runtime/sys.go:207:32: runtime.readLinesFromFile calls bufio.Reader.ReadLine, which eventually calls http2.gzipReader.Read
      #24: database/kv/etcd/v3/client.go:30:2: etcd.init calls client.init, which eventually calls http2.headerFieldNameError.Error
      #25: database/kv/etcd/v3/client.go:30:2: etcd.init calls client.init, which eventually calls http2.headerFieldValueError.Error
      #26: database/kv/etcd/v3/client.go:30:2: etcd.init calls client.init, which eventually calls http2.pseudoHeaderError.Error
      #27: log/color.go:100:14: log.CEPrintfln calls fmt.Fprintf, which eventually calls http2.stickyErrWriter.Write
      #28: database/kv/nacos/config_client.go:57:40: nacos.NacosConfigClient.newConfigClient calls clients.NewConfigClient, which eventually calls http2.transportResponseBody.Close
      #29: runtime/sys.go:207:32: runtime.readLinesFromFile calls bufio.Reader.ReadLine, which eventually calls http2.transportResponseBody.Read
      #30: container/gxlru/lru_cache.go:199:20: gxlru.LRUCache.StatsJSON calls fmt.Sprintf, which eventually calls http2.writeData.String
**What happened**:

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

--- a/go.mod
+++ b/go.mod
@@ -20,7 +20,7 @@ require (
 	go.etcd.io/etcd/server/v3 v3.5.0-alpha.0
 	go.uber.org/atomic v1.9.0
 	go.uber.org/zap v1.21.0
-	golang.org/x/crypto v0.0.0-20211108221036-ceb1ce70b4fa
+	golang.org/x/crypto v0.21.0
 	google.golang.org/grpc v1.56.3
 	gopkg.in/yaml.v2 v2.4.0
 )
@@ -69,10 +69,10 @@ require (
 	go.etcd.io/etcd/pkg/v3 v3.5.0-alpha.0 // indirect
 	go.etcd.io/etcd/raft/v3 v3.5.0-alpha.0 // indirect
 	go.uber.org/multierr v1.6.0 // indirect
-	golang.org/x/net v0.9.0 // indirect
+	golang.org/x/net v0.23.0 // indirect
 	golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4 // indirect
-	golang.org/x/sys v0.7.0 // indirect
-	golang.org/x/text v0.9.0 // indirect
+	golang.org/x/sys v0.18.0 // indirect
+	golang.org/x/text v0.14.0 // indirect
 	golang.org/x/time v0.0.0-20220722155302-e5dcc9cfc0b9 // indirect
 	google.golang.org/genproto v0.0.0-20230410155749-daa745c078e1 // indirect
 	google.golang.org/protobuf v1.30.0 // indirect

